recipes/ignition-tools/all/conandata.yml

```diff
@@ -0,0 +1,20 @@
+sources:
+  #"1.0.0": 
```

Comment:
please, remove the old versions and keep only 1.4.0. In case you really need an older version, you can keep, but it should be built. If any other need an older version, someone can open a new PR adding it. Less is more.